### ISSUE TITLE
imaginary-teleprompter.desktop: missing semicolon (';') as trailing character

### ISSUE THREAD

**Issue by probonopd** [imaginaryfilms/Teleprompter-Electron#1]
`desktop-file-validate` says:

`imaginary-teleprompter.desktop: error: value "Video" for string list key "Categories" in group "Desktop Entry" does not have a semicolon (';') as trailing character`

Reference:
https://travis-ci.org/AppImage/appimage.github.io/builds/266238276#L543

**Comment by Cuperino** [comment]
Hello @probonopd,

Thank's for submitting the issue and for suggesting using `desktop-file-validate`. On what package and version of Teleprompter did you find this?

I'm using version 2.3.1, the .pacman package (since I'm running Manjaro), and looking at the .desktop file I find the semicolon to be present. `desktop-file-validate` shows:
`hint: value item "Sequencer" in key "Categories" in group "Desktop Entry" can be extended with another category among the following categories: AudioVideo;Audio`,
which I find a little strange because we're extending from Video, which extends from AudioVideo, and this should be ok to do: `AudioVideo;Video;Sequencer;`.

Nevertheless, the issue you bring may have been present on a previous version of Teleprompter. Please make sure you're running the latest version, and report back whether the issue still occurs on your distribution, stating: version, package, and distribution. Thanks!

**Comment by probonopd** [comment]
> On what package and version of Teleprompter did you find this?

 https://travis-ci.org/AppImage/appimage.github.io/builds/266238276#L537

**Comment by Cuperino** [comment]
That's version 2.0's AppImage. I've never used Travis before. Was this version selected automatically by Travis as the "latest release"? Version 2.0 uses has version tag "v2.0", while later version tags do not include the letter 'v' in them. I suspect that may have confused the system. Anyway, the correct paths to our latest two releases are:
- 2.3.1 https://sourceforge.net/projects/teleprompter-imaginary-films/files/v2.3.1/imaginary-teleprompter-2.3.1.AppImage/download
- 2.3.0 https://github.com/ImaginarySense/Teleprompter-Electron/releases/download/2.3/imaginary-teleprompter-2.3.0.AppImage (has bug that prevents scrolling from reaching the end in some use-cases.)

I'll try to update the "v2.0" tag without changing the branch the release points to. As of 2.3.1 we were publishing our releases only to SourceForge and hyperlinking them from GitHub Releases, in order to have unified downloads counts. Please let me know if this affects Travis deployment in any way.

**Comment by Cuperino** [comment]
I've fixed the tag versioning issue. Please let me know if there's anything else that needs to be done for Travis to compile. Thanks.

**Comment by probonopd** [comment]
Why are there some links to SourceForge on the GitHub Releases page? This makes it much, much harder to automatically grab the latest download. Do you still need SourceForge, is there a reason you can't entirely switch to GitHub?

**Comment by Cuperino** [comment]
We started uploading binaries to SourceForge only in order to avoid diluting download counts across multiple sites. It helps potential users discover our software if we're both at SourceForge and GitHub. We hope to later join Linux repositories, as well and create snaps to keep expanding our user base.

I wasn't aware until now that removing the binaries from GitHub would affect automation. We'll go back to uploading binaries to both GitHub and SourceForge in the hopes of favoring build automation and Travis users.

I just uploaded the latest binaries to the GitHub release. Test them out and let us know how it works.
